mne/commands/mne_watershed_bem.py

```diff
@@ -0,0 +1,114 @@
+"""
+    Create BEM surfaces using the watershed algorithm included with
+        FreeSurfer
+
+"""
+
+from __future__ import print_function
+import sys
+import os
+import shutil
+import mne
+
+
+def mne_watershed_bem(subject, subjects_dir, overwrite=False, volume='T1', atlas=False, gcaatlas=False, preflood=None):
```

Comment:
subjects_dir should be None by default. If None use os.environ and raise error if not available.

also run pep8 on this file.